src/Groups/homomorphisms.jl

```diff
@@ -371,6 +371,33 @@ function isisomorphic(G::GAPGroup, H::GAPGroup)
   end
 end
 
+
+################################################################################
+#
+#  Conversions between types
+#
+################################################################################
+
+_get_iso_function(::Type{PermGroup}) = GAP.Globals.IsomorphismPermGroup
+_get_iso_function(::Type{FPGroup}) = GAP.Globals.IsomorphismFpGroup
+_get_iso_function(::Type{PcGroup}) = GAP.Globals.IsomorphismPcGroup
+
+
+"""
+    isomorphism(::Type{T}, G::GAPGroup) where T <: Union{FPGroup, PcGroup, PermGroup}
+
+Return an isomorphism from `G` to a group of type `T`.
+An exception is thrown if no such isomorphism exists.
+"""
+function isomorphism(::Type{T}, G::GAPGroup) where T <: Union{FPGroup, PcGroup, PermGroup}
```

Comment:
Should we perhaps cache these isomorphisms, i.e.
```suggestion
@attr function isomorphism(::Type{T}, G::GAPGroup) where T <: Union{FPGroup, PcGroup, PermGroup}
```

Reply:
Caching is a good idea, but `isomorphism` takes two arguments, and `@attr` can deal only with unary functions.
(From this point of view, the old `isomorphic_fp_group` etc. were more suitable.)

Reply:
Yeah, we can't just use `@attr`, but we can still "manually" implement that caching. Anyway, not in this PR.